### PR TITLE
Feature/show app name on hover

### PR DESCRIPTION
**Pull request by erik-sn**
Enables tooltips on app names and adds some TS interfaces around app configurations.

### CODE REVIEW

**Review by jfrader — APPROVED** [review]
Nice work!

**Comment by itschip** [comment]
LGTM 👍

**Review by itschip — APPROVED** [review]

**Comment by TasoOneAsia** [comment]
Wait a second does this pass when the DOM is resized @erik-sn?

I had previously implemented this but when zoom property is changed the portal coordinates don't recalculate properly

File: phone/src/ui/components/GridMenu.tsx [hunk 1]
```diff
@@ -35,25 +43,23 @@ export const GridMenu = ({ items, Component = AppIcon }) => {
...
-            //   title={item.id}
-            //   placement='top'
-            //   arrow
-            //   classes={{ tooltip: classes.tooltip }}
-            //   PopperProps={{
-            //     anchorEl={}
-            //   }}
-            // >
-            <Grid
+            <Tooltip
```
Comment (TasoOneAsia) [inline phone/src/ui/components/GridMenu.tsx:46]: Portals are broken when using the CSS zoom property, as React's underlying render is not updated to match the dimensions of the DOM element in which the property `zoom` is applied to. See the following image for an example:

Dom Layout

**Comment by TasoOneAsia** [comment]
As a result of these breaking changes, we should probably revert the commit, what do you guys think? @erik-sn @itschip @jfrader

**Comment by RockySouthpaw** [comment]
I tested this in game and can confirm with taso that unless the zoom is set to 100%, this doesn't work as it doesn't scale with various zoom levels.

**Comment by erik-sn** [comment]
Yep can be reverted, I didn't know about the zoom issue. I will take a more in depth look at that.

**Pull request merged**